course/src/main/java/in/testpress/course/util/SwipeToDeleteCallback.kt

```diff
@@ -0,0 +1,103 @@
+package `in`.testpress.course.util
+
+import `in`.testpress.course.ui.OfflineExamListActivity
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.graphics.drawable.Drawable
+import android.view.View
+import androidx.recyclerview.widget.ItemTouchHelper
+import androidx.recyclerview.widget.RecyclerView
+
+class SwipeToDeleteCallback(
+    private val offlineExamAdapter: OfflineExamListActivity.OfflineExamAdapter,
```

Comment:
This says its a util but we are hardcoding here as OfflineExamAdapter.
This is not a util, this can be only used for OfflineExamAdapter. Make it usable to any Adapter